config.example.json

```diff
@@ -1,25 +1,25 @@
 {
-  "server": {
-    "host": "mc.hypixel.net",
-    "port": 25565
-  },
-  "minecraft": {
-    "username": "yourUsername",
-    "password": "yourPassword",
-    "lobbyHolder": "ignUsername",
-    "accountType": "mojang"
-  },
-  "discord": {
-    "token": "discordBotToken",
-    "channel": "channelId",
-    "commandRole": "roleId",
-    "ownerId": "ownerId",
-    "prefix": "!",
-    "messageMode": "bot"
-  },
-  "express": {
-    "enabled": false,
-    "port": 8880,
-    "authorization": "authorizationHeaderString"
-  }
-}
+    "server": {
+      "host": "mc.hypixel.net",
+      "port": 25565
+    },
+    "minecraft": {
+      "username": "literally_anything_here",
+      "lobbyHolder": "Not_Sure_About_This_One",
+      "accountType": "microsoft"
```

Comment:
To force microsoft accounts this should be removed from the config and hard coded into where the mineflayer bot is created rather than put as microsoft in the example.